paper/mitchell-geophysics.tex

```diff
@@ -0,0 +1,934 @@
+
+Electrical and possibly electromagnetic (EM) methods offer the best chances of imaging the wet zones as a result of the large contrast in conductivity between the wet salt or briny water (0.1 - 20 S/m) and dry salt ($2.5 \times 10^{-5}$ - 0.01 S/m) \citep{Duckworth1992, Chouteau1997}. Identifying the location and geometry of these water bearing regions is vitally important to mining operations to avoid flooding. Accurate conductivity models of the region surrounding the mine workings can help the engineers identify and seal off current water inflow sources and hopefully avoid hitting water bearing zones in the future.
+
+\subsection{DC Resistivity Data and Instrumentation}
+\label{CaseStudy:Data_Instrumentation}
+The data used in this study were collected using a non-conventional 3D dipole-dipole array, which was designed and collected by Golder Associates. The full survey uses 120 stainless steel electrodes which are combined in a variety of combinations to form a survey comprised of 2,351 current electrode pairs (transmitters, TXs) and 95,194 associated potential electrode pairs (receivers, RXs). The measurements were collected using an IRIS Syscal Pro, which is a multi-channel switching system capable of simultaneously measuring 10 potential differences for a given TX (current dipole). Switching cables were 50 m long with electrode take-outs every 5 m. Six switching cables are linked with connecting boxes to connect electrodes 1-60 with the control box and another six switching cables are used to connect electrodes 61-120. We refer to these combined cables as Cable 1 and Cable 2 respectively.
+
+The geometry of the tunnels and the general layout of the electrodes are shown in Figure \ref{fig:DataQC_OctreeMesh_ElecGeom} while a typical TX is shown in Figure \ref{fig:DataQC_TypicalTx}. Within the mine, electrode placement is typically restricted to the mine workings (drifts-horizontal tunnels, raises-small vertical shafts, and smaller sub-drifts). Main drifts are typically 3-6 m wide and 3-4 m tall, most of the raises extend approximately 20 m vertically up to the Second Red Bed (Shale) above the Prairie Evaporite, and the sub-drifts are usually 2-3 m wide and 2-3 m tall. The electrodes are laid out along the interior tunnel walls as shown in Figure \ref{fig:DataQC_OctreeMesh_ElecGeom} with a spacing of 5 m to image the central region between the tunnels, where a water inflow source is suspected.
+
+\begin{figure} [!ht]
+    \begin{center}
+    \subfigure[Discretized tunnel and electrode geometry]{%

```

Comment:
myst question... is there a plan to have `subfigure`'s recognized? or what would your recommendation be for how we handle these? Manually combine the images?

Reply:
This is a new type of sub figure! We have \begin{subfigure} completed. Haven't seen this variant before. Will extend things!